lib/stripe/stripe_client.rb

```diff
@@ -31,11 +31,11 @@ def self.default_conn
       # of connection re-use, so make sure that we have a separate connection
       # object per thread.
       Thread.current[:stripe_client_default_conn] ||= begin
-        conn = Faraday.new do |c|
-          c.use Faraday::Request::Multipart
-          c.use Faraday::Request::UrlEncoded
-          c.use Faraday::Response::RaiseError
-          c.adapter Faraday.default_adapter
+        conn = Faraday.new do |builder|
```

Comment:
Changed this variable name to `builder` to be more accurate and descriptive as to what's being received inside the block here.